### PR TITLE
[REF] Improve signature on updateFinancialAccountsOnContributionStatusChange

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Minor function cleanup

Before
----------------------------------------
Params being calculated in parent function that could be calculated in called function

After
----------------------------------------
Params not calculated in calling function

Technical Details
----------------------------------------

Comments
----------------------------------------

### CODE REVIEW

File: CRM/Contribute/BAO/Contribution.php [hunk 1]
```diff
@@ -1101,13 +1101,14 @@ protected static function cancel($processContributionObject, $memberships, $cont
...
-   * @param array $previousContributionStatus
-   * @param string $currentContributionStatus
    *
    * @return bool[]
    *   Return indicates whether the updateFinancialAccounts function should continue & whether this is a refund.
    */
-  private static function updateFinancialAccountsOnContributionStatusChange(&$params, $context, $previousContributionStatus, $currentContributionStatus) {
+  private static function updateFinancialAccountsOnContributionStatusChange(&$params, $context) {
+    $previousContributionStatus = CRM_Contribute_PseudoConstant::contributionStatus($params['prevContribution']->contribution_status_id, 'name');
+    $currentContributionStatus = CRM_Core_PseudoConstant::getName('CRM_Contribute_BAO_Contribution', 'contribution_status_id', $params['contribution']->contribution_status_id);
```
Comment (mattwire) [inline CRM/Contribute/BAO/Contribution.php]: @eileenmcnaughton This is wierd - we are using two different pseudoconstant functions to get the statuses - can we switch the other one to getName() as part of this PR?

**Review by mattwire — CHANGES_REQUESTED** [review]

File: CRM/Contribute/BAO/Contribution.php (hunk 1 above)
Comment (eileenmcnaughton) [reply CRM/Contribute/BAO/Contribution.php]: @mattwire can we not scope creep this one - it's moving existing lines

Comment (eileenmcnaughton) [reply CRM/Contribute/BAO/Contribution.php]: @mattwire per comment - I'd rather just move & not change within this Pr

**Comment by eileenmcnaughton** [comment]
unrelated fails

**Pull request merged**